app/views/catalog/_home_text.html.erb

```diff
@@ -13,26 +13,36 @@ Unless required by applicable law or agreed to in writing, software distributed
   specific language governing permissions and limitations under the License.
 ---  END LICENSE_HEADER BLOCK  ---
 %>
-<h2 id="featured_content_header">Sample Content</h2>
+<h2 id="featured_content_header">Featured Content</h2>
 <div id="featured_content" class="row">
   <div class="col-md-4">
-    <h4>Featured Collection</h4>
-    <a href="#">
-      <%= image_tag "Nutcracker-Ballet.png", alt: "Featured Collection" %>
+    <h4>Featured Video Collection</h4>
+    <a href="/catalog?f%5Bcollection_ssim%5D%5B%5D=Nosotros+Television+Programme&q=&search_field=all_fields" class="embed-responsive embed-responsive-16by9">
+      <%= video_tag "welcome0.mp4", autoplay: 'true', alt: "Featured Video Collection", class: "embed-responsive-item"%>
+      <div class="caption post-content"><h5>Nosotros Television Programme</h5></div>
     </a>
   </div>
   <div class="col-md-4">
-    <h4>Featured Video</h4>
-     <a href="#">
-        <%= image_tag "Charlie-McCarthy.png", alt: "Featured Collection" %>
-      </a>
+    <h4>Featured Audio Collection</h4>
+    <a href="/catalog?f%5Bavalon_resource_type_ssim%5D%5B%5D=Sound+Recording&f%5Bcollection_ssim%5D%5B%5D=Convocation+Hall+" class="embed-responsive embed-responsive-16by9">
+      <%= video_tag "welcome2.mp4", autoplay: 'true', alt: "Featured Audio Collection", class: "embed-responsive-item"%>
```

Comment:
I'm quite hesitant of having 3/4 auto playing videos on the home page? (which is probably not a great idea to begin with from a user perspective, as it's overwhelming, especially if they got sound playing)

But from a website point of view, the 3/4 videos almost total up to 5MBs, which is a huge load on the homepage?

(I say 3/4 videos cause nothing is referencing `welcome3.mp4`? So probably added by accident?)

 Which is a pretty big chunk for a user to download (aka slows your webpage down considerably, think of someone on mobile phone or something....ouch).

Also what about our bandwidth? For example just getting 1000 views on avalon is 5GBs down (5MB per request for just videos, times 1000). Now lets assume we get 100k views a month?

Reply:
@umarqasim could you advise on how you'd like the autoplay to be handled?

Reply:
I don't want any of them to play, just wanted to use a static frame. We had these in stock(purchased earlier) so I thought may be we use them and these look good too. If it is not possible to use one frame, we can convert the last frame to an image. Let me know if you want me to convert it.